VideoLocker/src/main/java/org/edx/mobile/view/CourseVideoListActivity.java

```diff
@@ -122,90 +119,39 @@ protected void modeChanged(){
-                    mHandler.removeCallbacks(mHideRunnable);
-                    if (downloadIndicator.getAnimation() == null) {
-                        downloadIndicator.setVisibility(View.VISIBLE);
-                        Animation animation = AnimationUtils.loadAnimation(CourseVideoListActivity.this, R.anim.rotate);
-                        downloadIndicator.startAnimation(animation);
-                    }
-                } else { //progressPercent == 100
-                    downloadIndicator.clearAnimation();
-                    downloadIndicator.setVisibility(View.INVISIBLE);
-                    mHandler.postDelayed(mHideRunnable, getResources().getInteger(R.integer.message_delay));
-                }
-            }
-        }, 500);
-    }
-
-    @Override
-    protected void setVisibilityForDownloadProgressView(boolean show){
-         boolean visible = downloadProgressBar.getVisibility() == View.VISIBLE;
-        if (visible == show )
-            return; //do nothing
-
-        if ( show ){
-            //TODO - we pass a value less than 100 to indicate it is downloading.
-            updateDownloadProgress(0);
-        } else {
-            updateDownloadProgress(100);
-        }
-    }
+    protected void updateDownloadProgress(final int progressPercent){}
 
     @Override
     public void onDownloadSuccess(Long result) {
-        try {
-            updateListUI();
-        }catch(Exception e){
-            logger.error(e);
-        }
+        updateListUI();
     }
 
     @Override
     public void onDownloadFailure() {
+        updateListUI();
     }
 
     @Override
-    public void showProgressDialog(int numDownloads) {
-        setVisibilityForDownloadProgressView(true);
-    }
+    public void showProgressDialog(int numDownloads) {}
 
     @Override
     public abstract  void updateListUI();
 
-
-    private Runnable mHideRunnable = new Runnable() {
-        @Override
-        public void run() {
-            downloadProgressBar.setVisibility(  View.GONE );
-        }
-    };
-
     //TODO - legacy code use one minute tick loop to sync some UI status, like
     //total download progress. this is a simple approach, but may not be the
     //best one.
     protected void onTick() {
         // this is a per second callback
         try {
-                if(AppConstants.offline_flag){
-                    setVisibilityForDownloadProgressView(false);
-                }else{
-                    if(environment.getDatabase()!=null){
-                        boolean downloading = environment.getDatabase().isAnyVideoDownloading(null);
-                        if(!downloading){
-                            setVisibilityForDownloadProgressView(false);
-                        }else{
-                            environment.getStorage().getAverageDownloadProgress(averageProgressCallback);
-                        }
-                    }   //store not null check
-                }
-            }  catch(Exception ex) {
```

Comment:
I think since we're no longer using this to power the row animations, this method can be reduced to a stub override as well.